host/CMakeLists.txt

```diff
@@ -342,6 +342,21 @@ list(
   strings.c
   traceh_enclave.c)
 
+# Add MSVC compile flag /Z7 to enable debugging into OE SDK source from
```

Comment:
Instead of doing this, you can just use `target_compile_options` against the `oehost` target.  See an example at: https://github.com/openenclave/openenclave/blob/e8f54a6b5c146eca81eb8c740f645674c6ec6a36/host/CMakeLists.txt#L485

Also, with these, we don't need the special handling for calls.c as it's part of `PLATFORM_SDK_ONLY_SRC`.

Reply:
When I added /z7 to oehost, it was throwing invalid compile flag error for sgx/windows/aep.asm. Hence had to add flag to all .c srcs.

Reply:
Removed special assignment of \z7 flag to host/sgx/calls.c